vignettes/tidytranscriptomics.Rmd

```diff
@@ -937,34 +928,7 @@ We can easily summarise the results. For example, we can see how cell type class
 ```{r}
 pbmc %>% count(label, first.labels)
 ```
-## Pseudobulk analyses
-
-It is sometime useful to aggregate cell-wise transcript abundance into pseudobulk samples. It is possible to explore data and perform hypothesis testing with tools and data-source that we are more familiar with. For example, we can use edgeR in tidybulk to perform differential expression testing. For more details on pseudobulk analysis see [here](https://hbctraining.github.io/scRNA-seq/lessons/pseudobulk_DESeq2_scrnaseq.html).
-
-### Data exploration using pseudobulk samples
-
-To do this, we will load a helper function called `aggregate_cells` from [GitHub Gist](https://gist.github.com/stemangiola/ebce5a3f931b298611b56680d39c073c) and use it to create a group for each sample.
-
-```{r warning=FALSE, message=FALSE}
-# Load aggregate function
-devtools::source_gist("889e4089a0e5ca2649d3c9164e3b542e")
-
-# Aggregate
-pbmc_bulk = 
-  bioc2021tidytranscriptomics::pbmc %>% 
-  aggregate_cells(file) 
-
-pbmc_bulk
-```
-
-```{r}
```

Comment:
Hello @mblue9 , maybe we can eliminate this after having duplicated the repository for Tunis workshop (I can do that), so in this repository we would have it , and we can always easily drop it at a second stage.

Reply:
ok